### PR TITLE
chore: upgrade to pnpm 9

### PR DESCRIPTION
**Pull request by aqrln**
Update the driver adapters tests to use pnpm 9 to match the client repo.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -405,7 +405,7 @@ build-driver-adapters-kit: build-driver-adapters
 
 build-driver-adapters: ensure-prisma-present
 	@echo "Building driver adapters..."
-	@cd ../prisma && pnpm --filter "*adapter*" i
+	@cd ../prisma && pnpm i
```
Comment (jkomyno) [inline Makefile:408]: Is this a needed change? I imagine it'll increase the CI time

**Review by jkomyno — APPROVED** [review]
Approved with a comment

File: Makefile (hunk 1 above)
Comment (aqrln) [reply Makefile:408]: Yes, it was failing even after updating to pnpm 9 in this PR precisely because of this.

Comment (aqrln) [reply Makefile:408]: (and it was also the reason i couldn't reproduce locally, because i already had all packages installed in the local checkout)

**Pull request merged**